ADRs/README.md

```diff
@@ -72,8 +72,10 @@ To contribute a new ADR, please follow these steps:
 3. Fill in the necessary information in the new ADR file, following the template structure.
 4. Use the large language model prompts, if desired, to generate or refine the ADR content.
 5. Submit a pull request to merge your ADR branch into the main branch, including a brief description of the problem and solution.
-6. Request a review from the team members, and address any feedback or suggestions provided.
-7. Once approved, merge the pull request to update the repository with the new ADR.
+6. Add a reference to the [ADR Index](https://www.notion.so/pocketnetwork/1a5893d2183b470b93610d2d9ce84d36?v=91b83d5b44d34ba6abf36f9070c2f3a8&pvs=4) for your pull request with appropriate status.
```

Comment:
Cool! :raised_hands: 

Just curious what the rationale originally was for this? I can see value in having a view of all the ADRs in one place for multiple reasons / use cases.
 
I find that it makes the ADRs more browsable because it presents **all** ADRs in one place - improved user experience. Using GitHub UI alone automatically bifurcates the ADRs which can bee browsed together into those which have and have not progressed past the "accepted" state:
  - Browsing in GitHub and the repo itself will only present ADRs which have passed the "in review" state (e.g. "accepted", "withdrawn", "superseded").
  - Applying specific filters in the GitHub pull requests page is likely the most reliable way to collect ADRs which are "in review" or earlier in the lifecycle (e.g. "draft") for browsing.

Maintaining that index does introduce a risk of it falling out of sync; however, I think the benefit outweighs the additional overhead at this point and I also think it's likely that we'll be able to largely (or completely) automate it in future process iterations.

Lastly, my interpretation of the share settings is that this is not currently public to the community; was that intentional?:
![image](https://github.com/pokt-network/pocket-network-protocol/assets/600733/9a2cbfd6-71c3-4c3f-b1eb-ba24246134ad)

Did you intend to toggle the "share to web" toggle?

Reply:
The original rationale was to have all ADRs in a single place for visibility even after closure of the pull request. There is danger in the two falling out of sync but the alternative is to keep them withdrawn ADRs in a folder.  I'm open to that, but this felt lightweight for now.

I absolutely did mean to share to web, I will do that now 💯